operator/src/main/java/oracle/kubernetes/operator/DomainStatusUpdater.java

```diff
@@ -408,6 +413,29 @@ DomainStatus getNewStatus() {
       if (newStatus.getMessage() == null) {
         newStatus.setMessage(info.getValidationWarningsAsString());
         if (existingError != null) {
+
+          boolean isFromModel = Optional.ofNullable(info)
+              .map(DomainPresenceInfo::getDomain)
+              .map(Domain::getDomainHomeSourceType)
+              .equals(DomainSourceType.FromModel);
+
+          if (isFromModel) {
+            boolean onlineUpdate = Optional.ofNullable(info)
+                .map(DomainPresenceInfo::getDomain)
+                .map(Domain::getSpec)
+                .map(DomainSpec::getConfiguration)
+                .map(Configuration::getModel)
+                .map(Model::getOnlineUpdate)
+                .map(OnlineUpdate::getEnabled)
+                .orElse(false);
+            if (onlineUpdate) {
```

Comment:
Just because onlineUpdate is enabled, is that the only possible reason we've reached this point in the code? Or could other failures unrelated to onlineUpdate have brought us here?